scipy/linalg/_matfuncs_expm.pyx.in

```diff
@@ -0,0 +1,691 @@
+# cython: language_level=3
+cimport cython
+import math
+import random
+cimport numpy as cnp
+import numpy as np
+from libc.stdlib cimport malloc, free
+from libc.math cimport fabs, ceil, log2, pow
+from scipy.linalg.cython_lapack cimport (sgetrf, sgetrs, dgetrf, dgetrs,
+                                         cgetrf, cgetrs, zgetrf, zgetrs)
+from scipy.linalg.cython_blas cimport (sgemm, saxpy, sscal, scopy, sgemv,
+                                       dgemm, daxpy, dscal, dcopy, dgemv,
+                                       cgemm, caxpy, cscal, ccopy, cgemv,
+                                       zgemm, zaxpy, zscal, zcopy, dgemv,
+                                       csscal, zdscal, idamax)
+
```

Comment:
Looks like a `cnp.import_array()` is missing (see gh-14813).

Reply:
That's a good catch indeed. Now added.